pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -656,6 +671,9 @@ func (oe *operationExecutor) generateMountVolumeFunc(
 		mountErr := volumeMounter.SetUp(fsGroup)
 		if mountErr != nil {
 			// On failure, return error. Caller will log and retry.
+			if errGetRef == nil && ref != nil {
```

Comment:
I tried passing in an event with nil ref and it didn't work, it doesn't error, but the event never shows up